scripts/uberenv/packages/raja/package.py

```diff
@@ -287,8 +288,12 @@ def hostconfig(self, spec, prefix, py_site_pkgs_dir=None):
                                         hip_root))
             cfg.write(cmake_cache_entry("HIP_CLANG_PATH",
                                         rocm_root + '/llvm/bin'))
-            cfg.write(cmake_cache_entry("HIP_HIPCC_FLAGS",
-                                        '--amdgpu-target=gfx906'))
+
+            hipcc_flags = '--amdgpu-target=gfx906'
+            if "+desul" in spec:
+                hipcc_flags += ';-std=c++14'
```

Comment:
This confuses me. The ";" isn't a typo, is it?

Reply:
Oh, that enables the flags to be added together, I assume? Hmm lol that seems dangerous :p